metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/v3/controller/EntityController.java

```diff
@@ -497,7 +497,13 @@ public ResponseEntity<Map<String, List<GenericEntityV3>>> createGenericEntities(
     }
```

Comment:
```suggestion
    String entityType = result.getUrn().toLowerCase();
```

Let's change this above to have this just be consistent